File: Maths_And_Stats/Geometry/Radians_Degrees_Conversion/Radians_Degrees_Conversion.js
const pi = 3.1415926535;

// function to convert degrees to radians
const degreesToRadians = (degrees) => {
  return degrees * (pi / 180);
};

// function to convert radians to degrees
const radiansToDegrees = (radians) => {
  return radians * (180 / pi);
};

//driver code
const degreeInput = prompt("Enter a number to convert from degree to radian: ");
let radianOutput = degreesToRadians(degreeInput);
console.log(`${degreeInput}° = ${radianOutput} rad`);

const radianInput = prompt("Enter a number to convert from radian to degree: ");
let degreeOutput = radiansToDegrees(radianInput);
console.log(`${radianInput} rad = ${degreeOutput}°`);

File: Maths/Geometry/Radians_Degrees_Conversion/Radians_Degrees_Conversion.cpp
#include <iostream> 
#include <math.h> 
using namespace std; 
  
// Function for conversion 
double Convert(double degree) 
{ 
    double pi = 3.14159265359; 
    return (degree * (pi / 180)); 
} 
  
// Driver code 
int main() 
{ 
    double degree;
    cout<<"Enter a value: <<endl;
    cin>>degree;
    double radian = Convert(degree); 
    cout << radian; 
    return 0; 
} 
